### PR TITLE
WIP: Small improvements and typo corrections.

### PR DESCRIPTION
**Pull request by flavianojs**

### CODE REVIEW

**Comment by flavianojs** [comment]
@mbercx do you know why these commits are failing the continuous-integration tests?

**Comment by mbercx** [comment]
> @mbercx do you know why these commits are failing the continuous-integration tests?

These were already failing from my initial push, it seems. 😅  I just wanted to have a working html build so you could start going through the material. Let me see if I can fix these issues in a separate branch and then you can rebase.

PS: Let me know when the PR is ready for review!

File: docs/index.rst [hunk 1]
```diff
@@ -3,22 +3,18 @@ AiiDA ABC tutorial
...
 
-.. todo::
-
-   Update versions to ones from Quantum Mobile image
-
 +-----------------+-----------------------------------------------------------------------------------------------------------+
 | Related resources                                                                                                           |
 +=================+===========================================================================================================+
-| python packages | `aiida-core XXXX`_, `aiida-quantumespresso XXXX`_                                                         |
+| python packages | `aiida-core 1.3.0`_, `aiida-quantumespresso 3.0.0`_                                                       |
```
Comment (mbercx) [inline docs/index.rst:9]: The package versions probably won't correspond to these. I'm assuming they will still be the same as those from the BIGMAP tutorial.

```suggestion
| python packages | `aiida-core 1.4.3`_, `aiida-quantumespresso 3.2.1`_                                                       |
```

File: docs/index.rst [hunk 2]
```diff
@@ -3,22 +3,18 @@ AiiDA ABC tutorial
...
 +-----------------+-----------------------------------------------------------------------------------------------------------+
-| codes           | `Quantum ESPRESSO XXXX`_                                                                                  |
+| codes           | `Quantum ESPRESSO 6.5`_                                                                                   |
```
Comment (mbercx) [inline docs/index.rst:11]: Same here, unfortunately. I left these as XXXX's to fix when the AiiDAlab cluster was up and running. On https://aiidalab-demo.materialscloud.org/ the Quantum ESPRESSO version is still 6.0.

File: docs/source/sections/data.rst [hunk 3]
```diff
@@ -4,7 +4,11 @@
 Working with data and querying your results
 *******************************************
 
-In this section of the tutorial we will focus on how to organise and explore the data in an AiiDA database, based on a previously created database  for this tutorial.
+.. todo::
+
+   Probably requires update.
```
Comment (mbercx) [inline docs/source/sections/data.rst:9]: So you didn't test this section yet, right? 🙃

**Review by mbercx — CHANGES_REQUESTED** [review]
Thanks @flavianojs! Just left some minor comments, mainly re the versioning. I think these will actually match those of the BIGMAP tutorial. I'll test the whole querying section  on the [AiiDAlab demo cluster](https://aiidalab-demo.materialscloud.org/) after this is merged. Tomorrow we can give the whole tutorial one final pass on the actual cluster we'll be using.

File: docs/source/sections/data.rst (hunk 3 above)
Comment (mbercx) [reply docs/source/sections/data.rst:9]: Just quickly executed the commands in the section and then ran the whole solution notebook. Seems we have to remove the duplicate groups made with this command:

```console
$ verdi group copy tutorial_lda tutorial/lda/basic
$ verdi group copy tutorial_pbe tutorial/gga/pbe
$ verdi group copy tutorial_pbesol tutorial/gga/pbesol
```

Else the final plot won't work (it expects 3 pseudopotential groups, not 6).

File: docs/index.rst (hunk 1 above)
Comment (mbercx) [reply docs/index.rst:9]: Just checked the versions, and these are correct!

**Pull request merged**